packages/test/service-load-test/src/nodeStressTest.ts

```diff
@@ -150,22 +169,56 @@ async function runnerProcess(
     testId: string,
 ): Promise<number> {
     try {
+        const beganMs = Date.now();
         const runConfig: IRunConfig = {
             runId,
             testConfig: profile,
+            leaveDelayMs: randomIntBetween(15000, 25000),
         };
+        // TODO: consider moving "magic" numbers to JSON file
+        const offlineChance = 0.375;
+        const offlineMs = 2 * runConfig.leaveDelayMs;
+        const joinDelayMs = randomIntBetween(4500, 2500);
+        const joinLeaveCycleGoal = 5;
+        let joinLeaveCycles = 0;
+        let offlineLimit = 1;
 
         const testDriver = await createTestDriver(driver);
+        const reconnect = createReloader(testDriver, testId);
+
+        do {
+            let offlineDelayMs = 0;
+
+            if (offlineLimit > 0 && randomBetween(0, 1) < offlineChance) {
+                offlineLimit -= 1;
+                offlineDelayMs = offlineMs;
+            }
+
+            const stressTestP = reconnect(joinDelayMs + offlineDelayMs);
+            if (offlineDelayMs) {
+                stdout("offline", `for ${Math.floor(0.001 * offlineDelayMs)}s`);
+            }
+            const stressTest = await stressTestP;
 
-        const stressTest = await load(testDriver, testId);
-        await stressTest.run(runConfig);
-        console.log(`${runId.toString().padStart(3)}> exit`);
+            await stressTest.run(runConfig);
+        } while (++joinLeaveCycles < joinLeaveCycleGoal);
+
+        const runtimeSeconds = Math.floor(0.001 * (Date.now() - beganMs));
+        stdout("exit", `after ${runtimeSeconds}s`);
         return 0;
     } catch (e) {
-        console.error(`${runId.toString().padStart(3)}> error: loading test`);
+        stdout("error", "loading test");
         console.error(e);
         return -1;
     }
+
+    function stdout(tag: string, message: any = undefined) {
+        console.error(
```

Comment:
You mean `console.log`, right? Or are you wanting it to be highlighted differently?